web/ASC.Web.Common/src/components/MediaViewer/MediaViewer.js

```diff
@@ -0,0 +1,318 @@
+import { Icons } from "asc-web-components";
+
+import ImageViewer from "./sub-components/image-viewer"
+import VideoViewer from "./sub-components/video-viewer"
+import MediaScrollButton from "./sub-components/scroll-button"
+import ControlBtn from "./sub-components/control-btn"
+import isEqual from "lodash/isEqual";
+
+const StyledVideoViewer = styled(VideoViewer)`
+    z-index: 4001;
+`
+const StyledMediaViewer = styled.div`
+    
+    color: #d1d1d1;
+    display: ${props => props.visible ? "block" : "none"};
+    overflow: hidden;
+    .videoViewerOverlay{
+        position: fixed;
+        z-index: 4000;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        background: black;
+        opacity: 0.5;
+    }
+    .mediaViewerToolbox{
+        z-index: 4001;
+        padding-top: 14px;
+        padding-bottom: 14px;
+        height: 20px;
+        width: 100%;
+        background-color: rgba(11,11,11,0.7);
+        position: fixed;
+        bottom: 0;
+        left: 0;
+        text-align: center;
+    }
+    span{
+        position: fixed;
+        right: 0;
+        bottom: 5px;
+        margin-right: 10px;
+        z-index: 4005;
+
+        .deleteBtnContainer{
+            display: block;
+            width: 20px;
+            margin: 3px 10px;
+            line-height: 19px;
+        }
+
+        .downloadBtnContainer{
+            display: block;
+            width: 20px;
+            margin: 3px 10px;
+            line-height: 19px;
+        }
+    }
+    .details{
+        z-index: 4001;
+        font-size: 14px;
+        font-weight: bold;
+        text-align: center;
+        white-space: nowrap;
+        padding-top: 14px;
+        padding-bottom: 14px;
+        height: 20px;
+        width: 100%;
+        background: rgba(17,17,17,0.867);
+        position: fixed;
+        top: 0;
+        left: 0;
+    }
+
+    .mediaPlayerClose{
+        position: fixed;
+        top: 4px;
+        right: 10px;
+        height: 30px;
+    }
+  
+`;
+
+var audio = 1;
+var video = 2;
+class MediaViewer extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            visible: this.props.visible,
+            allowConvert: true,
+            playlist: this.props.playlist,
+            playlistPos: 0,
+        };
+
+        this.detailsContainer = React.createRef();
+        this.viewerToolbox = React.createRef();
+    }
+
+    componentDidUpdate(prevProps) {
+        if (this.props.visible !== prevProps.visible) {
+            this.setState(
+                {
+                    visible: this.props.visible,
+                    playlistPos: this.props.playlist.length > 0 ? this.props.playlist.find(file => file.fileId === this.props.currentFileId).id : 0
+                }
+            );
+        }
+        if (this.props.visible == true && this.props.visible === prevProps.visible && !isEqual(this.props.playlist, prevProps.playlist)) {
```

Comment:
Replace "this.props.visible == true" to "this.props.visible". Type of visible prop is boolean in PropTypes